File: src/main/java/leets/leenk/domain/leenk/domain/service/LeenkDeleteService.java
package leets.leenk.domain.leenk.domain.service;

import leets.leenk.domain.leenk.domain.entity.Leenk;
import leets.leenk.domain.leenk.domain.repository.LeenkRepository;
import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Service;

@Service
@RequiredArgsConstructor
public class LeenkDeleteService {

    private final LeenkRepository leenkRepository;

    public void delete(Leenk leenk) {
        leenkRepository.delete(leenk);
    }
}

File: src/main/kotlin/leets/leenk/domain/leenk/domain/service/LeenkDeleteService.kt
package leets.leenk.domain.leenk.domain.service

import leets.leenk.domain.leenk.domain.entity.Leenk
import leets.leenk.domain.leenk.domain.repository.LeenkRepository
import org.springframework.stereotype.Service

@Service
class LeenkDeleteService(
    private val leenkRepository: LeenkRepository,
) {
    fun delete(leenk: Leenk) {
        leenkRepository.delete(leenk)
    }
}
